sample/android-sdk-sample/app/src/main/java/com/bitmark/androidsdksample/MainActivity.java

```diff
@@ -0,0 +1,79 @@
+package com.bitmark.androidsdksample;
+
+import android.app.FragmentTransaction;
+import android.os.Bundle;
+import android.support.design.widget.BottomNavigationView;
+import android.app.Fragment;
+import android.support.v7.app.AppCompatActivity;
+
+import com.bitmark.androidsdksample.constants.AppConstants;
+import com.bitmark.androidsdksample.fragments.AccountFragment;
+import com.bitmark.androidsdksample.fragments.QueryFragment;
+import com.bitmark.androidsdksample.fragments.RegistrationIssuanceFragment;
+import com.bitmark.androidsdksample.fragments.TransferFragment;
+import com.bitmark.androidsdksample.samples.SDKSample;
+
+public class MainActivity extends AppCompatActivity {
+    private BottomNavigationView navView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        setupBottomNavigation();
+
+        if (savedInstanceState == null) {
+            loadAccountFragment();
+        }
+
+        SDKSample.initialize(AppConstants.API_TOKEN, AppConstants.NETWORK_MODE);
+    }
+
+    private void setupBottomNavigation() {
+        navView = findViewById(R.id.nav_view);
+
+        navView.setOnNavigationItemSelectedListener(item -> {
```

Comment:
I think you can simplify the code with the variable to hold the `Fragment` will be replaced and assign it in each case of `item.getItemId()` then call method to replace fragment at the end. 
Add `default` case to just `return false`